.travis.yml

```diff
@@ -24,4 +24,9 @@ deploy:
     file: "/home/travis/${image_name}"
     on:
       tags: true
-      branch: master
+    
+after_deploy:
+  - ./mail.py
+
+branch: master
```

Comment:
Don't you think we should change this to 
`all_branches: true`
for the deployment in every branch since we are going to use generator branch for config.